azurerm/resource_arm_monitor_action_group_test.go

```diff
@@ -440,38 +930,186 @@ resource "azurerm_monitor_action_group" "test" {
   resource_group_name = "${azurerm_resource_group.test.name}"
   short_name          = "acctestag"
 
-  email_receiver {
-    name          = "sendtoadmin"
-    email_address = "admin@contoso.com"
-  }
+  automation_runbook_receiver {
+		name = "action_name_1"
+		automation_account_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-runbooks/providers/microsoft.automation/automationaccounts/aaa001"
+		runbook_name = "my runbook"
+		webhook_resource_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-runbooks/providers/microsoft.automation/automationaccounts/aaa001/webhooks/webhook_alert"
+		is_global_runbook = true
+		service_uri = "https://s13events.azure-automation.net/webhooks?token=randomtoken"
+		use_common_alert_schema = false
+	}
+}
+`, rInt, location, rInt)
+}
 
-  email_receiver {
-    name          = "sendtodevops"
-    email_address = "devops@contoso.com"
-  }
+func testAccAzureRMMonitorActionGroup_voiceReceiver(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
 
-  sms_receiver {
-    name         = "oncallmsg"
-    country_code = "1"
-    phone_number = "1231231234"
-  }
+resource "azurerm_monitor_action_group" "test" {
+  name                = "acctestActionGroup-%d"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  short_name          = "acctestag"
 
-  sms_receiver {
-    name         = "remotesupport"
-    country_code = "86"
-    phone_number = "13888888888"
-  }
+	voice_receiver {
+		name         = "oncallmsg"
+		country_code = "1"
+		phone_number = "1231231234"
+	}
 
-  webhook_receiver {
-    name        = "callmyapiaswell"
-    service_uri = "http://example.com/alert"
-  }
+	voice_receiver {
+		name         = "remotesupport"
+		country_code = "86"
+		phone_number = "13888888888"
+	}
+}
+`, rInt, location, rInt)
+}
 
-  webhook_receiver {
-    name        = "callmybackupapi"
-    service_uri = "https://backup.example.com/warning"
-  }
+func testAccAzureRMMonitorActionGroup_logicAppReceiver(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_monitor_action_group" "test" {
+  name                = "acctestActionGroup-%d"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+	short_name          = "acctestag"
+
+	logic_app_receiver {
+		name = "logicappaction"
+		resource_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg-logicapp/providers/Microsoft.Logic/workflows/logicapp"
```

Comment:
(same here) could we dynamically provision a logic app workflow and use it's ID here?